### PR TITLE
Add Contour 0.2.0.173

### PR DESCRIPTION
**Pull request by christianparpart**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)? (NOTE: Yes, I think, because I used `wingetcreate` to create the yaml files)

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24985)

### CODE REVIEW

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hey there @christianparpart, the `PackageVersion` needs to match the version number in `A`dd or `R`emove `P`rograms
image
And since the version is `0.2.0 RC1` this would count as a newer version than `0.2.0` when you release that.

For now i would advice you to do one of these
* Just Submit `0.1.1` instead so its in the Repo for when `0.2.0` is ready for release.
* Change this version to `0.2.0` if its ready for people to use and update the `InstallerUrl` and `InstallerSha256` to the `0.2.0` metadata when its released.
* Change the `PackageIdentifier` and folder structure to `Contour.Terminal.RC` aka `manifests/c/Contour/Terminal/RC/0.2.0 RC1/`, the PackageName should also be `Contour RC` i this case so the people on the Stable version don't accidently update.
* Wait for Channel support (no ETA) so Stable/RC/Beta/Alpha/Dev can be in the same folder.

**Comment by christianparpart** [comment]
Oh wie. Many many thanks for the detailed response. I will actually release 0.2.0 then as it's only minors that are known and can easily go into 0.2.1 later then. I will do that ASAP. Many thanks.

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Gonna slap a `Requested Changes` on this PR so it gets the `Needs: author feedback` label so it doesn't get picked up in the review list, convert this into a draft or update the PR whenever the version is released :)

**Comment by christianparpart** [comment]
Let's see if it all now passes.... (don't auto-close too early, i'm also busy IRL with family-business. trying to react ASAP always) ^^

File: manifests/c/Contour/Terminal/0.2.0.173/Contour.Terminal.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Copyright: Copyright 2019-2021 by Christian Parpart
+ShortDescription: Contour Terminal Emulator
+Description: Contour is a modern terminal emulator, for everyday use. It is aiming for power users with a modern feature mindset.
+Moniker: contour
+Tags:
+- terminal
+- console
+- command line
+- wsl
+- developer tools
```
Comment (ItzLevvie) [inline manifests/c/Contour/Terminal/0.2.0.173/Contour.Terminal.locale.en-US.yaml:24]:
```suggestion
- command-line
- wsl
- developer-tools
```

File: manifests/c/Contour/Terminal/0.2.0.173/Contour.Terminal.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: Contour.Terminal
+PackageVersion: 0.2.0.173
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/contour-terminal/contour/releases/download/v0.2.0.173/contour-0.2.0.173-win64.msi
+  InstallerSha256: 092E6952C7BEDE41D8044CBA7078D52E9F16E5B915B5FF457A6C5EF053DF0B7C
+  ProductCode: '{E92B15F2-0C08-485C-9888-A6C4A14EC8F0}'
```
Comment (ItzLevvie) [inline manifests/c/Contour/Terminal/0.2.0.173/Contour.Terminal.installer.yaml:12]:
```suggestion
  ProductCode: '{88234011-14C8-4A62-8962-CCBF1B7DD62D}'
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/c/Contour/Terminal/0.2.0.173/Contour.Terminal.installer.yaml (hunk 2 above)
Comment (christianparpart) [reply manifests/c/Contour/Terminal/0.2.0.173/Contour.Terminal.installer.yaml:12]: I forgot where i have got the first Baker from. How did your get that? Is this changing over every version / MSI creation?

Comment (OfficialEsco) [reply manifests/c/Contour/Terminal/0.2.0.173/Contour.Terminal.installer.yaml:12]: It does seem to change for every MSI creation (now that i think about not for Firefox), i believe the developer chooses the ProductCode for exe installers (or the default might be the AppName/Moniker)

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/24985#pullrequestreview-734811167

**Comment by christianparpart** [comment]
```pwsh
wmic product where "Name like '%Contour%'" get Name, Version, IdentifyingNumber
```

I used that command to get the ProductCode. I am not so sure that's the most effecient way to get the code (and I hope I'm doing it right).

**Comment by christianparpart** [comment]
p.s.: i just realized by now that there is a test failing. will fix that. (writing the message so the PR won't get auto-closed).

File: manifests/c/Contour/Terminal/0.2.0/Contour.Terminal.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Contour.Terminal
+PackageVersion: 0.2.0 RC1
```
Comment (ItzLevvie) [inline manifests/c/Contour/Terminal/0.2.0/Contour.Terminal.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.2.0.173
```

File: manifests/c/Contour/Terminal/0.2.0/Contour.Terminal.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Contour.Terminal
+PackageVersion: 0.2.0 RC1
```
Comment (ItzLevvie) [inline manifests/c/Contour/Terminal/0.2.0/Contour.Terminal.installer.yaml:5]:
```suggestion
PackageVersion: 0.2.0.173
```

File: manifests/c/Contour/Terminal/0.2.0/Contour.Terminal.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Contour.Terminal
+PackageVersion: 0.2.0 RC1
```
Comment (ItzLevvie) [inline manifests/c/Contour/Terminal/0.2.0/Contour.Terminal.yaml:5]:
```suggestion
PackageVersion: 0.2.0.173
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by christianparpart** [comment]
Oh dear. I was child-sick and will repost simply ASAP. :-)

**Pull request closed without merging**